docs/src/events.md

```diff
@@ -164,6 +195,11 @@ page.once("dialog", lambda dialog: dialog.accept("2021"))
 page.evaluate("prompt('Enter a number:')")
 ```
 
+```csharp
+page.Dialog += (_, dialog) => dialog.AcceptAsync("2021");
```

Comment:
This is a section about once listener - do we even have that in C#?

Reply:
We don't